pkgs/applications/video/video-trimmer/default.nix

```diff
@@ -0,0 +1,93 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, rustPlatform
+, gnome
+, pkg-config
+, meson
+, wrapGAppsHook4
+, appstream-glib
+, desktop-file-utils
+, blueprint-compiler
+, ninja
+, python3
+, gtk3-x11
+, glib
+, gobject-introspection
+, gtk4
+, libadwaita
+, gst_all_1
+}:
+
+stdenv.mkDerivation rec {
+  pname = "video-trimmer";
+  version = "0.7.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "YaLTeR";
+    repo = "video-trimmer";
+    rev = "v${version}";
+    sha256 = "sha256-D7wjJkdqqjjwwYEUZnNr7hFQK59wfTnaCLXCy+SK8Jo=";
+  };
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-cB5dVrEbISvHrOb87uVZSkT694VKtPtyk+c1tYNCTp0=";
+  };
+
+  patches = [
+    # The metainfo.xml file has a URL to a screenshot of the application,
+    # unaccessible in the build's sandbox. We don't need the screenshot, so
+    # it's best to remove it.
+    ./remove-screenshot-metainfo.diff
```

Comment:
The proper solution is passing `--nonet` argument to `appstream-util`. Or you can just not add `appstream-glib` since it is not required:

https://gitlab.gnome.org/YaLTeR/video-trimmer/-/blob/b2bdcdc8eb2112e38059dad2abdae2b32015b17e/data/meson.build#L38-43